superset/jinja_context.py

```diff
@@ -213,6 +176,142 @@ def url_param(
             self.cache_key_wrapper(result)
         return result
 
+    def filter_values(
+        self, column: str, default: Optional[str] = None, remove_filter: bool = False
+    ) -> List[str]:
+        """Gets a values for a particular filter as a list
+
+        This is useful if:
+            - you want to use a filter component to filter a query where the name of
+             filter component column doesn't match the one in the select statement
+            - you want to have the ability for filter inside the main query for speed
+            purposes
+
+        Usage example::
+
+            SELECT action, count(*) as times
+            FROM logs
+            WHERE
+                action in ({{ "'" + "','".join(filter_values('action_type')) + "'" }})
+            GROUP BY action
+
+        :param column: column/filter name to lookup
+        :param default: default value to return if there's no matching columns
+        :param remove_filter: When set to true, mark the filter as processed,
+            removing it from the outer query. Useful when a filter should
+            only apply to the inner query
+        :return: returns a list of filter values
+        """
```

Comment:
Could we add a deprecation warning here, urging to migrate to `get_filters` and stating that this function will be removed in 2.0? A similar warning can be found here:

https://github.com/apache/superset/blob/a9d888ad402ebb35da45df446997c426d6abee9d/superset/app.py#L422-L425